tsdb/wal/checkpoint.go

```diff
@@ -181,6 +184,24 @@ func Checkpoint(logger log.Logger, w *WAL, from, to int, keep func(id uint64) bo
 			stats.TotalSeries += len(series)
 			stats.DroppedSeries += len(series) - len(repl)
 
+		case record.Metadata:
+			metadata, err = dec.Metadata(rec, metadata)
+			if err != nil {
+				return nil, errors.Wrap(err, "decode series")
```

Comment:
series?

Reply:
Ah good call, yes will change this to "decode metadata".